src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrUpdater.cs

```diff
@@ -653,6 +659,43 @@ private void DeleteRepository(string repo)
         }
     }
 
+    /// <summary>
+    /// This method is called in cases when a repository is already at the target revision but the package version
+    /// differs. This can happen when a new build from the same commit is synchronized to the VMR.
+    /// </summary>
+    private async Task UpdateTargetVersionOnly(
+        string targetRevision,
+        string targetVersion,
+        SourceMapping mapping,
+        VmrDependencyVersion currentVersion,
+        CancellationToken cancellationToken)
+    {
+        _logger.LogWarning(
```

Comment:
Should this be a warning? It's not a case we see very often, but it's still something that's a pretty normal thing

Reply:
Yeah, I imagine we'd want to learn about this in the build. We were just chasing this down so I think this should be visible